manifests/plugin/openscap/params.pp

```diff
@@ -6,4 +6,7 @@
   $listen_on               = 'https'
   $openscap_send_log_file  = '/var/log/foreman-proxy/openscap-send.log'
   $spooldir                = '/var/spool/foreman-proxy/openscap'
+  $contentdir              = '/usr/share/foreman-proxy/openscap/content'
```

Comment:
It depends on what you do with it, but foreman-proxy should not write to ```/usr/share/foreman-proxy```. reports implies these are the uploaded reports from clients. You would expect them to be somewhere in ```/var``` (and ```/usr/share/foreman-proxy/openscap/reports``` could be a symlink.